src/renderer/viz/expressions/base.js

```diff
@@ -207,4 +207,8 @@ export default class Base {
     eval(feature) {
         throw new Error('Unimplemented');
     }
+
+    isA(expressionClass) {
```

Comment:
This PR looks ✨ The only thing, I would name this function `isExpression` instead of `isA`, because `isA` is not very self explanatory 😂 I would be great to add a test as well.

Reply:
I'm sorry you don't like my `IsA` 😢  It has a [long tradition](https://en.wikipedia.org/wiki/Is-a) in OO.

Anyway, let's see 🤔 I'm not sure what naming I prefer...
* `expr.isA(Property)`
* `expr.isExpression(Property)`
* `expr.kindOf(Property)`
* `expr.instanceOf(Property)`
* `expr.isExpressionOfType(Property)`

I'm not sure! 😕

Reply:
If it is a tradition/convention, then is ok to use `isA` :)

Reply:
hahahaha, just saying... one file - multiple classes is also a tradition in CARTO VL :smile_cat:

Reply:
:trollface: